### PR TITLE
[2.1] Clarify effect of ratio updates on centeredness

### PR DESCRIPTION
**Pull request by EndymionJkb**
# Description

(BALA1-4) Document edge case where large pools with low fees doing rapid price ratio changes have some level of MEV.

This tests the boundaries and provides concrete guidance to governance.

## Type of change

- [ ] Bug fix 
- [ ] New feature 
- [ ] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Optimization: [ ] gas / [ ] bytecode
- [X] Documentation or wording changes
- [ ] Other

## Checklist:

- [X] The diff is legible and has no extraneous changes
- [N/A] Complex code has been commented, including external interfaces
- [X] Tests have 100% code coverage
- [X] The base branch is either `main`, or there's a description of how to merge

### CODE REVIEW

**Review by jubeira — APPROVED** [review]
LGTM, makes sense.

File: test/foundry/ReClammSwap.t.sol [hunk 1]
```diff
@@ -458,3 +459,179 @@ contract ReClammLongIdleDrainTest is BaseReClammTest {
...
 }
+
+/**
+ * @notice Documents the MEV surface around in-range price ratio widening updates.
+ * @dev When a price ratio update is active, `computeVirtualBalancesUpdatingPriceRatio` preserves the pool's current
+ * centeredness at the new (wider) ratio. Because centeredness is computed from live balances, a swap during the
+ * update window changes the centeredness that gets locked in, and the widening amplifies the attacker-chosen
+ * imbalance.
+ *
+ * Whether this is profitable depends on pool depth: the extraction scales with pool size (wider pool = less
```
Comment (joaobrunoah) [inline test/foundry/ReClammSwap.t.sol:470]: Price ratio and swap fee percentage, right? More real liquidity increases depth, but doesn't make the pool wider.

File: test/foundry/ReClammSwap.t.sol [hunk 2]
```diff
@@ -458,3 +459,179 @@ contract ReClammLongIdleDrainTest is BaseReClammTest {
...
+        assertGt(attackerAAfter, attackerABefore, "Round trip should profit at large pool size with zero fees");
+
+        // The profit should be material. The audit's witness shows ~2.55% of the swap amount.
+        uint256 profit = attackerAAfter - attackerABefore;
+        uint256 profitBps = (profit * 10000) / swapAmount;
+        assertGt(profitBps, 200, "Profit should be at least 2% of swap amount");
+    }
+
+    /// @dev Same large-pool scenario, but with a swap fee. Searches for the fee that neutralizes the profit.
+    function testWideningRoundTripNeutralizedByFees() public {
```
Comment (joaobrunoah) [inline test/foundry/ReClammSwap.t.sol:542]: Cool!

**Review by joaobrunoah — APPROVED** [review]
LGTM!
